src/Sylius/Bundle/ApiBundle/Command/Cart/PickupCart.php

```diff
@@ -36,9 +29,8 @@ class PickupCart implements ChannelCodeAwareInterface, CustomerEmailAwareInterfa
     /** @var string|null */
     public $email;
```

Comment:
Why these three fields left as standalone fields?

Reply:
These fields are not initialized in the constructor